hazelcast-sql/src/main/java/com/hazelcast/jet/sql/impl/connector/map/LazyDefiningSpecificMemberPms.java

```diff
@@ -54,13 +56,11 @@ public LazyDefiningSpecificMemberPms() {
     private LazyDefiningSpecificMemberPms(ProcessorSupplier supplier, int partitionArgumentIndex) {
         this.supplier = supplier;
```

Comment:
is the variant with `partitionArgumentIndex` an optimisation for `ParameterExpression` case?

Reply:
> is the variant with partitionArgumentIndex an optimisation for ParameterExpression case?
> if so, do we expect significant benefit?

Yes, it's the simpler option, when you know an exact column and don't want to wrap expression into the supplier.